definitions.yaml

```diff
@@ -596,6 +596,33 @@ definitions:
               awsoperator:
                 type: string
                 description: ARN of the IAM role to assume by the software operating clusters
+      azure:
+        type: object
+        description: Credentials specific to an Azure service principal
+        required:
+          - credential
+        properties:
+          credential:
+            type: object
+            descriptions: Service principal credential
+            required:
+              - clientID
+              - secretID
+              - subscriptionID
+              - tenantID
+            properties:
+              clientID:
+                type: string
+                description: Client ID of the service principal
+              secretID:
```

Comment:
should `secretID` be called `secretKey` ?

Reply:
true, that's no ID. I think there might be more places in different repo where to fix this

Reply:
fixed